src/editor.js

```diff
@@ -168,6 +162,9 @@ export default function Editor({
       buffer.select_range(selectStartIterator, selectEndIterator);
     });
   });
+
+  messageText.grab_focus();
+
```

Comment:
There is a regression at least on `main` but the intention is for the `Cancel` button to have the focus. 

This way user can use `Tab` to navigate to `Commit` or text editing.

Reply:
From a UX perspective, wouldn't it make more sense to have the text focused? Users can cancel with Esc; We could also add a shortcut for Ctrl+Enter.  For tab navigation there's Ctrl+Tab.

For most cases `Commit` wouldn't be sensitive, so it's redundant to  give `Cancel` focus.

Reply:
https://github.com/sonnyp/Commit/issues/33